src/element.ts

```diff
@@ -44,8 +44,8 @@ export abstract class Element {
   protected style?: ElementStyle;
   private attrs: ElementAttributes;
   protected boundingBox?: BoundingBox;
-  protected fontStack: Font[];
-  protected musicFont: Font;
+  protected fontStack!: Font[];
+  protected musicFont!: Font;
```

Comment:
Please add a comment saying that they are initialized by SetFonts called by `constructor.